biome.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.6.4/schema.json",
+  "organizeImports": {
+    "enabled": true
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "a11y": {
+        "useValidAriaProps": "warn",
+        "useValidAriaValues": "warn",
+        "noAriaUnsupportedElements": "warn",
+        "useAriaPropsForRole": "warn",
+        "useAltText": "warn",
```

Comment:
Where do warnings show up locally?

Reply:
idk, I just copied these rules from eslint-config-next

Reply:
Can we figure out what command should be added to the scripts in package.json, so that folks can actually make use of these?

Reply:
I think these rules show up in your IDE if you have the Biome [add on](https://biomejs.dev/reference/vscode/) installed and the Biome daemon running. No extra script needed. The rules have to be changed explicitly because next‘s config differs from the standard Biome config